doc/provider/jumpcloud/README.md

```diff
@@ -0,0 +1,215 @@
+# saml2aws Documentation for JumpCloud
+
+Instructions for setting up single sign on (SSO) with Amazon AWS using
+[JumpCloud][1] and [saml2aws][2].
+
+---
+
+[](TOC)
+
+- [JumpCloud Single Sign On (SSO) with Amazon AWS in IAM](#jumpcloud-single-sign-on-sso-with-amazon-aws-in-iam)
+    - [Generate a public certificate and private key pair](#generate-a-public-certificate-and-private-key-pair)
+    - [Configure the new application in JumpCloud](#configure-the-new-application-in-jumpcloud)
+    - [Configure the new application in AWS](#configure-the-new-application-in-aws)
+    - [Assign the new application to groups](#assign-the-new-application-to-groups)
+- [AWS Management Console access](#aws-management-console-access)
+- [AWS programmatic access](#aws-programmatic-access)
+    - [Configure ](#configure-)
+    - [Login ](#login-)
+    - [Use](#use)
+
+[](TOC)
+
+---
+
+## JumpCloud Single Sign On (SSO) with Amazon AWS in IAM
+
+Based on the [instructions from JumpCloud][3], we'll setup administrative access
+for our production AWS account. We can then grant this access to our operations
+team. We will eventually want to setup administrative access for our other
+accounts (dev, test, staging, etc) as well as access for additional roles:
+
+* We may want to grant our accounts payable team the access they need to pay
+  our AWS bill on each of our accounts
+* We may want to give our developers the ability to manage ec2 resources on our
+  non-production accounts
+
+### Generate a public certificate and private key pair
+
+Based on the [instructions from JumpCloud][4], we'll generate a public
+certificate and private key pair for administrative access to our production
+AWS account.
+
+> I believe it's possible to use the same certificate and key for all SSO
+> applications; however, I think it is best practice to generate a pair for
+> each application.
+
+Create `production.cnf`:
```

Comment:
I believe it's possible to use the same certificate and key for all SSO
applications; however, I think it is best practice to generate a pair for
each application.